frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts

```diff
@@ -236,7 +240,14 @@ export const entityFilterLogic = kea<entityFilterLogicType<BareEntity, EntityFil
         },
         setFilters: async ({ filters }) => {
             if (typeof props.setFilters === 'function') {
-                props.setFilters(toFilters(filters))
```

Comment:
This was calling funnelLogic's `setFilters`, as set in `Experiment.tsx` line 99, but then it just stopped connecting to it suddenly? @alexkim205

Reply:
Played around with this a bit and confirming setFilters doesn't get called on my end too. I think `funnelLogic` in `Experiment.tsx` isn't mounted or overridden by the time it's supposed to be called, but not sure what the root cause is here

Reply:
ignore this